features/remotepollerng/daemon/src/test/java/org/opennms/netmgt/remotepollerng/RemotePollerdIT.java

```diff
@@ -215,6 +215,11 @@ public void setUp() throws Exception {
         locationAwarePollerClient.setRpcTargetHelper(new RpcTargetHelper());
         locationAwarePollerClient.afterPropertiesSet();
 
+        System.setProperty("org.opennms.netmgt.remotepollerng.trackerRefreshRateLimit", "5");
```

Comment:
I'd move the `"org.opennms.netmgt.remotepollerng.trackerRefreshRateLimit"` part to a constant and refer to that instead.

Reply:
Ay